bundle/src/main/java/com/adobe/acs/commons/redirects/servlets/RewriteMapServlet.java

```diff
@@ -1,8 +1,9 @@
-/*
+/*-
+ * #%L
  * ACS AEM Commons Bundle
- *
+ * %%
  * Copyright (C) 2013 - 2024 Adobe
- *
+ * %%
```

Comment:
@YegorKozlov I'm not sure if that makes sense - or should rather https://github.com/Adobe-Consulting-Services/acs-aem-commons/blob/master/.codeclimate/header.txt be tuned instead? Also the other Java files do not seem to adhere to this pattern.

@kwin @joerghoh @davidjgonzalez WDYT?

Reply:
@krystiannowak  this `%%` thing does not make much sense to me, but I believe we've always had it, since the header check was introduced in https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/1345 back in 2018

> Also the other Java files do not seem to adhere to this pattern.

Yep. We have quite a bit of the header variations across the project.